python/ray/ml/config.py

```diff
@@ -1,8 +1,58 @@
-from typing import Dict, Any
+from dataclasses import dataclass
+from typing import Dict, Any, Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ray.tune.trainable import PlacementGroupFactory
 
 # Right now, RunConfig is just an arbitrary dict that specifies tune.run
 # kwargs.
 # TODO(xwjiang): After Tuner is implemented, define the schema
 RunConfig = Dict[str, Any]
 
 ScalingConfig = Dict[str, Any]
+
+
+@dataclass
+class _ScalingConfigDataClass:
+    """Configuration for scaling training.
+
+    num_workers (int): The number of workers (Ray actors) to launch.
```

Comment:
```suggestion
    num_workers: The number of workers (Ray actors) to launch.
```

Please remove types from docstrings per our style guide.